optuna_dashboard/package.json

```diff
@@ -47,19 +46,14 @@
     "@babel/core": "^7.23.9",
     "@babel/preset-env": "^7.23.9",
     "@testing-library/react": "^14.1.2",
```

Comment:
Can we remove this as well?

Reply:
Thank you for your suggestion, I've removed `@testing-library/react`.